packages/common/src/models/Analytics.ts

```diff
@@ -338,7 +338,8 @@ export enum Name {
   SEND_MESSAGE_REACTION_FAILURE = 'Send Message Reaction: Failure',
   MESSAGE_UNFURL_TRACK = 'Message Unfurl: Track',
   MESSAGE_UNFURL_PLAYLIST = 'Message Unfurl: Playlist',
-  TIP_UNLOCKED_CHAT = 'Unlocked Chat: Tip'
+  TIP_UNLOCKED_CHAT = 'Unlocked Chat: Tip',
+  CHAT_REPORT_USER = 'Report User: Chat'
```

Comment:
is the thinking that there's other places to report users?

Reply:
In the future yeah potentially. Open to suggestions for naming convention.